src/components/PaneDropdown.vue

```diff
@@ -21,18 +21,14 @@
       <div
         :class="menuClasses"
         class="s-pane-dropdown__menu"
-        @mouseup="onMenuClick"
         v-if="paneMenuOpen"
       >
         <slot v-if="custom"></slot>
-        <div v-else class="s-pane-dropdown__list">
+        <div v-else @mouseup="onMenuClick" class="s-pane-dropdown__list">
           <slot></slot>
         </div>
       </div>
     </transition>
-    <span v-if="!custom" ref="panelinks" class="s-pane-dropdown__slot-list">
```

Comment:
@joshua-larks i just removed this :man_shrugging: ... in the demo section, it didn't seem to break anything